python/src/aac/plugins/first_party/lsp_server/providers/document_link_provider.py

```diff
@@ -0,0 +1,52 @@
+"""Module for the Document Links Provider which handles requests for document links."""
+from os import path
+from pygls.server import LanguageServer
+from pygls.lsp.types import DocumentLinkParams, DocumentLink
+from pygls.uris import from_fs_path, to_fs_path
+
+from aac.lang.language_context import LanguageContext
+from aac.plugins.first_party.lsp_server.providers.locations import get_location_from_lexeme
+from aac.plugins.first_party.lsp_server.providers.lsp_provider import LspProvider
+
+
+class DocumentLinkProvider(LspProvider):
+    """Resolve the full path for AaC document references."""
+
+    def handle_request(self, language_server: LanguageServer, params: DocumentLinkParams) -> list[DocumentLink]:
+        """Return the document links for a given file."""
+        self.language_server = language_server
```

Comment:
Is there a reason to make this an instance variable rather than just passing it into the `get_document_links` function?

Reply:
Not necessarily, just following the pattern established in other providers.